### PR TITLE
Rename email frame to email theme

### PR DESCRIPTION
**Pull request by richardolsson**
## Description
This PR renames "EmailFrame" to "EmailTheme".

## Screenshots
None

## Changes
* Uses the /email_themes API instead of /email_frames (not yet deployed)
* Changes names of internal variables

## Notes to reviewer
Note that the API change has not been deployed yet so this won't work against the live dev API.

## Related issues
Undocumented

### CODE REVIEW

File: src/features/emails/types.ts [hunk 1]
```diff
@@ -165,7 +165,7 @@ export type BlockAttributes = {
   }>;
 };
 
-export type EmailFrame = {
+export type EmailTheme = {
   block_attributes?: BlockAttributes;
   css?: string;
   frame_mjml: MJMLJsonObject | null;
```
Comment (ziggabyte) [inline src/features/emails/types.ts:171]: I think `fram_mjml` should also be renamed to `theme_mjml!`

File: src/features/emails/utils/deliveryProblems.spec.ts [hunk 2]
```diff
@@ -38,6 +37,7 @@ const mockEmail = (emailOverrides?: Partial<ZetkinEmail>): ZetkinEmail => {
       id: 123,
       ...emailOverrides?.target,
     },
+    theme: { frame_mjml: null, id: 1 },
```
Comment (ziggabyte) [inline src/features/emails/utils/deliveryProblems.spec.ts:40]: here same thing, that `frame_mjml` is renamed to `theme_mjml`

**Review by ziggabyte — DISMISSED** [review]
From reading the code, it looks good except there is one property I think should be renamed also :)

File: src/features/emails/types.ts (hunk 1 above)
Comment (richardolsson) [reply src/features/emails/types.ts:171]: I disagree about this. IMO, a theme is three things: The frame, the styling and the default block attributes. The `frame_mjml` property is the MJML for the frame part of the theme. Before the CSS and the block attributes were introduced, this thing that we used to call the frame was only a frame, but then it became more, which is why I think it's good to have renamed it to theme. But I don't think we should rename that part that is actually a frame.

Are you convinced? 😊

Comment (ziggabyte) [reply src/features/emails/types.ts:171]: You have convinced med! :D

**Pull request merged**